hack/tekton_ci.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash 
+set -e -o pipefail
+
+declare GITHUB_USER GITHUB_TOKEN GITHUB_ORG GITHUB_REPO 
+
+# This script deploys Tekton CI on a local kind cluster
+# It create GitHub webhook and deploys Tekton CI pipeline and tasks
+
+USAGE="Usage:  tekton_ci.sh -u <github-user> -t <github-token> -o <github-org> -r <github-repo>"
+
+# Read command line options
+while getopts ":u:t:o:r:" opt; do
+  case ${opt} in
+    u )
+      GITHUB_USER=$OPTARG
+      ;;
+    t )
+      GITHUB_TOKEN=$OPTARG
+      ;;
+    o )
+      GITHUB_ORG=$OPTARG
+      ;;
+    r )
+      GITHUB_REPO=$OPTARG
+      ;;
+    \? )
+      echo "Invalid option: $OPTARG" 1>&2
+      echo 1>&2
+      echo $USAGE
+      exit 1
+      ;;
+    : )
+      echo "Invalid option: $OPTARG requires an argument" 1>&2
+      exit 1
+      ;;
+  esac
+done
+shift $((OPTIND -1))
+
+# Check input params
+if [ -z "$GITHUB_USER" ] || [ -z "$GITHUB_TOKEN" ] || [ -z "$GITHUB_ORG" ] || [ -z "$GITHUB_REPO" ] ; then
+  echo "Missing parameters"
+  echo $USAGE
+  exit 1
+fi
+
+
+# Deploy plumbing resources. Run from the root of your local clone
+# The sed command injects your fork GitHub org in the CEL filters
+kustomize build tekton/ci | \
+  sed -E 's/tektoncd(\/p[^i]+|\(|\/'\'')/'$GITHUB_ORG'\1/g' | \
+  kubectl create -f -
```

Comment:
This part of the script is a bit brittle (as it was in the original documentation), unfortunately I have not found a better way to adapt this. This is something we may work on to improve in the future. 
We could use some kind of template or perhaps kustomize to inject the GITHUB_ORG and GITHUB_REPO in the CEL filters.

Reply:
agree, will think about that.